src/views/CommunityUpdates/CommunityUpdatesView.tsx

```diff
@@ -142,10 +142,12 @@ export const CommunityUpdatesView: FC<CommunityUpdatesViewProps> = ({ entities,
             if (onSubmit) {
               const message = await onSubmit(values['community-update']).finally(() => setSubmitting(false));
               setStubMessageId(message?.id || null);
+              if (message?.id) {
+                resetForm({
+                  values: initialValues,
+                });
+              }
             }
-            resetForm({
```

Comment:
It's on the same form. I detected that when an error occurred (because of the length of the comment or because of any other issue) the form was resetted and the comment was lost.
This way the form will only be resetted if the comment is successfully saved